src/third_party/starboard/wpe/rpi/gyp_configuration.py

```diff
@@ -25,15 +24,24 @@
 # will show up in an error message when that fails.
 _UNDEFINED_BUILDROOT_HOME = '/UNDEFINED/BUILDROOT_HOME'
 
+_UNDEFINED_OE_HOME = '/UNDEFINED/OE_HOME'
+_UNDEFINED_OE_NATIVE = '/UNDEFINED/OE_NATIVE'
 
 class WpePlatformConfig(platform_configuration.PlatformConfiguration):
   """Starboard Raspberry Pi platform configuration."""
 
   def __init__(self, platform):
     super(WpePlatformConfig, self).__init__(platform)
     self.AppendApplicationConfigurationPath(os.path.dirname(__file__))
-    self.buildroot_home = os.environ.get('BUILDROOT_HOME', _UNDEFINED_BUILDROOT_HOME)
-    self.sysroot = os.path.realpath(os.path.join(self.buildroot_home, 'arm-buildroot-linux-gnueabihf/sysroot'))
+    self.build_home = os.environ.get('BUILDROOT_HOME', _UNDEFINED_BUILDROOT_HOME)
+    if self.build_home != _UNDEFINED_BUILDROOT_HOME:
+      self.sysroot = os.path.realpath(os.path.join(self.build_home, 'arm-buildroot-linux-gnueabihf/sysroot'))
+      self.toolchain = 'arm-buildroot-linux-gnueabihf-'
+    else:
+      self.build_home = os.environ.get('OE_HOME', _UNDEFINED_OE_HOME)
+      if self.build_home != _UNDEFINED_OE_HOME:
+        self.sysroot = os.path.realpath(os.path.join(self.build_home, ''))
+        self.toolchain = os.environ.get('OE_NATIVE', _UNDEFINED_OE_NATIVE)
```

Comment:
Not sure about OE platform. 
But only difference in OE and BUILDROOT is the sysroot path.
However build for BUILDROOT works fine with the changes.

Reply:
What I mean here is have some variables set both in BROOT and OE to a proper values and here just read and use values of these vars without doing
if BROOT
else OE

Reply:
Thanks @pawel-stanek-ml, So I need to variable in OE case since the staging and toolchain are in different location. I will try to use the same way for both BR and OE (two variables to handle staging and toolchain)

Reply:
Looks good to me now. Closing this.